ckan/lib/flask_cli.py

```diff
@@ -0,0 +1,84 @@
+# encoding: utf-8
```

Comment:
IIUC this file is no longer needed right?

Reply:
yes, it can be removed